s/frontend/components/namaka-comment/namaka-comment.ts

```diff
@@ -83,6 +89,15 @@ export class NamakaComment extends mixinStandard<{
 		reportUserModalView({modals, comment})
 	}
 
+
+	#promptDeleteModal = async () => {
+		const {modals} = this.context
+		const comment = this.#getComment()
+		this.#toggleDropDown()
+		const result = await deleteThreadModalView({modals, comment})
+		result && this.#archiveThisComment()
```

Comment:
technically speaking, this code will result in a "dropped promise", becuase the promise is not awaited or returned

also, i think this line would be better as

```js
if (result)
  await this.#archiveThisComment()
```